elements/urth-viz-table/urth-viz-table.html

```diff
@@ -100,6 +100,12 @@
                     type: Number,
                     value: function () {return 15;},
                     reflectToAttribute: true
+                },
+
```

Comment:
Please add documentation to this property. Also update the documentation on `selection` so it explains how it is tied to this property.